### PR TITLE
Convert JSON annotations to new format 🎉

### PR DESCRIPTION
**Pull request by karawoo**
Going forward, annotations will be managed in a CSV file (currently called `sage_controlled_vocabulary.csv`) and there is code to generate a JSON-LD representation of the data model, a JSON Schema file that can be used for validation, and a manifest (in Google Sheets). This satisfies objective 2 of the working group. Existing data models have been translated to fit this format.

This is a draft PR because some refactoring is underway to turn the code into a standalone package that we can use. We'll hold off on merging until that is complete.

### CODE REVIEW

**Comment by karawoo** [comment]
@milen-sage I think this may be ready to merge, and then more documentation can be added later. Do you agree?

**Comment by milen-sage** [comment]
Yes - that looks good now! As we update schematic and/or documentation we can have separate PRs.

**Comment by karawoo** [comment]
@milen-sage will you formally approve this?

File: README.md [hunk 1]
```diff
@@ -72,6 +100,6 @@ The steps to create a release are as follows:
 
 Generally, you will next want to update the [Synapse table](https://www.synapse.org/#!Synapse:syn10242922) that describes the annotations and powers the [AnnotationUI](https://shinypro.synapse.org/users/nsanati/annotationUI/):
 
-1. Run the script `update_annotations_table.R`
+1. ~~Run the script `update_annotations_table.R`~~ TODO: update this to work with new CSV format.
```
Comment (milen-sage) [inline README.md:103]: Good point - we might want to revisit these instructions.

**Review by milen-sage — APPROVED** [review]

**Pull request merged**